udata/core/topic/parsers.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, with_include_private=True):
         self.parser.add_argument("granularity", type=str, location="args")
         self.parser.add_argument("organization", type=str, location="args")
         self.parser.add_argument("owner", type=str, location="args")
+        self.parser.add_argument("featured", type=bool, location="args")
```

Comment:
We should use [flask_restx boolean](https://flask-restx.readthedocs.io/en/latest/api.html#flask_restx.inputs.boolean) I think.
I made the change for `/api/1/datasets` in https://github.com/opendatateam/udata/pull/3298

```suggestion
        self.parser.add_argument("featured", type=boolean, location="args")
```

Reply:
Nice! Looks like much better parsing. Should I migrate `include_private` in the same PR so that we have the same behavior on both booleans?

Reply:
Yeah, let's keep the private mayhem for later :p